package it.prova.gestioneordini.exception;

public class ArticoloConCategoriaAssegnataException extends RuntimeException{
	
	public ArticoloConCategoriaAssegnataException(String message) {
		super(message);
	}
}
